tui/locales/hi-in/features.sh

```diff
@@ -9,9 +9,9 @@ GUI KDE प्लाज्मा के शीर्ष पर चलने व
 
 सक्षम करने के लिए कृपया सुविधाएँ चुनें:
 "
-TITLE="Open Voice OS Installation - विशेषताएँ"
-SKILL_DESCRIPTION="डिफ़ॉल्ट OVOS कौशल लोड करें"
-EXTRA_SKILL_DESCRIPTION="अतिरिक्त OVOS कौशल लोड करें"
-GUI_DESCRIPTION="ग्राफिकल यूजर इंटरफेस (GUI) इस्तमाल करें"
+TITLE="ओपन वॉयस ओएस इंस्टालेशन - विशेषताएं"
+SKILL_DESCRIPTION="बुनियादी कार्यक्षमता के लिए आवश्यक ओपन वॉयस ओएस कौशल लोड करें"
+EXTRA_SKILL_DESCRIPTION="व्यापक अनुभव के लिए अतिरिक्त ओपन वॉयस ओएस कौशल (वैकल्पिक) सेट लोड करें"
```

Comment:
बेहतर अनुभव के लिए अतिरिक्त ओपन वॉयस ओएस कौशल (वैकल्पिक) सेट लोड करें। 

This is more easy to understand